### ISSUE TITLE
[USABILITY] Improve User Experience for Event Organizers to Upload Assets

### ISSUE THREAD

**Issue by rrtti** [streamethorg/streameth-platform#932]
**Describe the usability issue**
Some suggestions to improve the upload experience in the platform. I list the ones I think are important below.

**Actual and Expected user experience**

1. Allow description size to be bigger when uploading a video, as well as the possiblity for the user to edit and format the description in the same way they can do it when clicking on "Edit" button after the video is uploaded. 

<img width="459" alt="Screenshot 2024-09-05 at 14 01 28" src="https://github.com/user-attachments/assets/ac850a3d-61be-4f5f-8921-ddece43125c3">

It would be good if the user can add spaces, add/delete rows, include hyperlinks and format the text, as well as add tags. Alternatively, it would be good to have an info icon on both title and description to know what the limit is. 

2. Change the following text once the asset is ready to be uploaded, so it is clear what the user needs to do:

<img width="453" alt="Screenshot 2024-09-05 at 14 22 58" src="https://github.com/user-attachments/assets/35d04da3-fe75-4dfd-808f-07172ea1c28d">

3. Fix visibility on thumbnails after uploading a video: when uploading an asset, as soon as it appears on the video list, it would be good to see the thumbnail (see screenshot with issue below). This prevents the user to go to the Edit section, delete the uploaded one and upload the image again to be visible. 

<img width="263" alt="Screenshot 2024-09-05 at 15 44 58" src="https://github.com/user-attachments/assets/ebd05e0d-1bde-4dd5-8831-a6adb2a800ad">

5. Organise assets in a discretional manner: it would be really good if the user could organise videos on the organization's landing page if they aim to provide more visibility to some assets than others. This way, the user will be able to customise their page but also organise videos in time-order if the assets were uploaded in a different way.

6. It would be good to be able to create playlists for assets: in order to separate assets by event, sections, formats, etc.  

7. Instead of an "Go Back To Assets" button in the successful message window after uploading an asset, I would rephrase the button to "Upload a new Asset" (since to go back to assets you need to close this window).

<3

**Comment by Mario-SO** [comment]
We heavily improved the upload UX, you can check again.

As for playlists, we have that in mind, but it has low priority right now. Will work on it in the future though as some other users have showed interest on it.

Thumbnails should also be fixed now 😃
